src/unwind.cc

```diff
@@ -53,8 +53,15 @@ static bool is_ld(const std::string &path) {
 }
 
 static bool is_stack_complete(UnwindState *us) {
-  static constexpr std::array s_expected_root_frames{"_start"sv, "__clone"sv,
-                                                     "_exit"sv};
+  static constexpr std::array s_expected_root_frames{
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏ wondering why some of these are needed (eg. `_exit`)

Reply:
I'm honestly not sure about that one :-)